cypresse2e/cypress/support/commands.js

```diff
@@ -10,6 +10,11 @@ Cypress.Commands.add('pageContainsHeadingAndImage', (pageHeadingLocator, heading
     pageImage.should('be.visible');
 });
 
+Cypress.Commands.add('pageContainsHeading', (pageHeadingLocator, heading) => {
```

Comment:
What do you think about this refactored option:

` cy.get(pageHeadingLocator).should('be.visible').contains(heading);` ??

This is just like in my first comment in this review.

Reply:
Refactored